egs/swbd/s5c/conf/decode_online.config

```diff
@@ -0,0 +1,2 @@
+beam=15.0 # beam for decoding.  Was 13.0 in the scripts.
```

Comment:
It looks like you may have copied this from somewhere like WSJ.  This beam is too large- try setting it to 11.0 which is the one used in conf/decode.config. @pegahgh, please re-test the online decoding, this may affect the WERs.